src/components/Messages/Message.tsx

```diff
@@ -52,12 +55,19 @@ const Message: FC<MessageProps> = ({ conversationKey }) => {
     }
   }, [conversationKey, hasMore, messages, endTime]);
 
+  const showLoading = !missingXmtpAuth && (!profile || !currentProfile || !selectedConversation);
+
+  useEffect(() => {
+    if (profile) {
+      Leafwatch.track('Pageview', { path: PAGEVIEW.MESSAGE });
+      Leafwatch.track(`Conversation with ${profile.ownedBy}`);
```

Comment:
I think we should avoid uploading any user info here (even `profile.ownedBy`) if we can avoid it. I'm not sure how useful this info would be and it can lead to privacy or PMI leaks.